2024-05-05 13:58:50,950 INFO    MainThread:1302 [wandb_setup.py:_flush():76] Current SDK version is 0.16.6
2024-05-05 13:58:50,950 INFO    MainThread:1302 [wandb_setup.py:_flush():76] Configure stats pid to 1302
2024-05-05 13:58:50,950 INFO    MainThread:1302 [wandb_setup.py:_flush():76] Loading settings from /Users/ruthikkale/.config/wandb/settings
2024-05-05 13:58:50,950 INFO    MainThread:1302 [wandb_setup.py:_flush():76] Loading settings from /Users/ruthikkale/Documents/GitHub/StockX/single_stock/wandb/settings
2024-05-05 13:58:50,950 INFO    MainThread:1302 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-05-05 13:58:50,951 INFO    MainThread:1302 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-05-05 13:58:50,951 INFO    MainThread:1302 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-05-05 13:58:50,951 INFO    MainThread:1302 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-05-05 13:58:50,951 INFO    MainThread:1302 [wandb_init.py:_log_setup():521] Logging user logs to /Users/ruthikkale/Documents/GitHub/StockX/single_stock/wandb/run-20240505_135850-q0ugevhe/logs/debug.log
2024-05-05 13:58:50,951 INFO    MainThread:1302 [wandb_init.py:_log_setup():522] Logging internal logs to /Users/ruthikkale/Documents/GitHub/StockX/single_stock/wandb/run-20240505_135850-q0ugevhe/logs/debug-internal.log
2024-05-05 13:58:50,951 INFO    MainThread:1302 [wandb_init.py:_jupyter_setup():467] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x3081fa510>
2024-05-05 13:58:50,951 INFO    MainThread:1302 [wandb_init.py:init():561] calling init triggers
2024-05-05 13:58:50,951 INFO    MainThread:1302 [wandb_init.py:init():568] wandb.init called with sweep_config: {}
config: {}
2024-05-05 13:58:50,952 INFO    MainThread:1302 [wandb_init.py:init():586] re-initializing run, found existing run on stack: ivee79td
2024-05-05 13:58:50,954 INFO    MainThread:1302 [wandb_run.py:_finish():2064] finishing run rk42218/LSTM/ivee79td
2024-05-05 13:58:50,954 INFO    MainThread:1302 [wandb_run.py:_atexit_cleanup():2304] got exitcode: 0
2024-05-05 13:58:50,954 INFO    MainThread:1302 [wandb_run.py:_restore():2287] restore
2024-05-05 13:58:50,955 INFO    MainThread:1302 [wandb_run.py:_restore():2293] restore done
2024-05-05 14:00:12,674 INFO    MainThread:1302 [wandb_init.py:init():611] starting backend
2024-05-05 14:00:12,674 INFO    MainThread:1302 [wandb_init.py:init():615] setting up manager
2024-05-05 14:00:12,686 INFO    MainThread:1302 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2024-05-05 14:00:12,687 INFO    MainThread:1302 [wandb_init.py:init():623] backend started and connected
2024-05-05 14:00:12,710 INFO    MainThread:1302 [wandb_run.py:_label_probe_notebook():1299] probe notebook
2024-05-05 14:00:12,710 INFO    MainThread:1302 [wandb_run.py:_label_probe_notebook():1309] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-05-05 14:00:12,710 INFO    MainThread:1302 [wandb_init.py:init():715] updated telemetry
2024-05-05 14:00:12,821 INFO    MainThread:1302 [wandb_init.py:init():748] communicating run to backend with 90.0 second timeout
2024-05-05 14:00:14,561 WARNING MainThread:1302 [wandb_init.py:init():1203] interrupted
Traceback (most recent call last):
  File "/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/wandb/sdk/wandb_init.py", line 1181, in init
    run = wi.init()
          ^^^^^^^^^
  File "/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/wandb/sdk/wandb_init.py", line 751, in init
    result = run_init_handle.wait(
             ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/wandb/sdk/lib/mailbox.py", line 283, in wait
    found, abandoned = self._slot._get_and_clear(timeout=wait_timeout)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/wandb/sdk/lib/mailbox.py", line 130, in _get_and_clear
    if self._wait(timeout=timeout):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/wandb/sdk/lib/mailbox.py", line 126, in _wait
    return self._event.wait(timeout=timeout)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ruthikkale/anaconda3/lib/python3.11/threading.py", line 622, in wait
    signaled = self._cond.wait(timeout)
               ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ruthikkale/anaconda3/lib/python3.11/threading.py", line 324, in wait
    gotit = waiter.acquire(True, timeout)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
KeyboardInterrupt
